### ISSUE TITLE
Different next's versions between create-next-app@latest and starter example

### ISSUE THREAD

**Issue by jonanrvz** [vercel/next-learn#787]
When creating the app, next 14 is used, but the version of next used by the starter example is version 15 RC, replacing the initial configurations and making it difficult to understand the step-by-step tutorial

**Comment by zuher83** [comment]
Yes, it's true, and I think it doesn't make sense to learn on an RC.

**Comment by leerob** [comment]
When you say "creating the app", what are you referring to? Are you starting from the start example?

**Comment by zuher83** [comment]
Yes start with:
`npx create-next-app@latest nextjs-dashboard --example "https://github.com/vercel/next-learn/tree/main/dashboard/starter-example" --use-pnpm
`
it installs Nextjs version 15 with react version 19... For my part, I've modified the json package and put back the packages for version 14 and react 18, and removed the node_modules folder and .next folder to make the tutorial! It works for me, but I'm not sure some people do that!

**Comment by leerob** [comment]
It is intentional it starts with v15 and React v19. The RC here will soon be stable, so we're starting out teaching this way for beginners 😄 

https://nextjs.org/15-rc

**Comment by zuher83** [comment]
@leerob Yes I understand, it's the first time I'm using NextJS but I'm an old developer 😅 and when launching the project I had a lot of errors 😮 with the RC version. I like stability 😉but with a few manipulations, I was able to get NextJS 14 working and learn the fundamentals in a few hours! Thanks for the job this is great to understand and get started 👌

**Comment by leerob** [comment]
Could you share the errors you were seeing? Would love to take a look.

**Comment by zuher83** [comment]
I can't remember the errors, but when I can I'll try again.

**Comment by progital** [comment]
@leerob some errors in this issue https://github.com/vercel/next-learn/issues/802

**Comment by leerob** [comment]
This is now updated!
